tests/test_import.py

```diff
@@ -1,31 +0,0 @@
-from mock import Mock
-
-from tests.tools import AgentTestCase
-import azurelinuxagent.common.osutil as osutil
-import azurelinuxagent.common.dhcp as dhcp
-import azurelinuxagent.common.protocol as protocol
-import azurelinuxagent.pa.provision as provision
-import azurelinuxagent.pa.deprovision as deprovision
-import azurelinuxagent.daemon as daemon
-import azurelinuxagent.daemon.resourcedisk as resourcedisk
-import azurelinuxagent.daemon.scvmm as scvmm
-import azurelinuxagent.ga.exthandlers as exthandlers
-import azurelinuxagent.ga.monitor as monitor
-import azurelinuxagent.ga.remoteaccess as remoteaccess
-import azurelinuxagent.ga.update as update
-
-
-class TestImportHandler(AgentTestCase):
```

Comment:
Just curious what was this file for?

Reply:
Beats me :) I see no use for it and needs to be update when changing those methods. Since it provides no useful tests I decided to delete it instead of keeping it up to date.

Reply:
Beats me :) I see no use for it and needs to be update when changing those methods. Since it provides no useful tests I decided to delete it instead of keeping it up to date